IDE/XCODE-FIPSv5/user_settings.h

```diff
@@ -374,6 +432,7 @@ extern "C" {
 #undef WOLFSSL_SHA3
 #if 1
     #define WOLFSSL_SHA3
+    #define Sha3 wc_Sha3
```

Comment:
This should be handled in wolfssl/wolfcrypt/sha3.h as long as NO_OLD_WC_NAMES is not defined?

Reply:
sha3.h gets replaced by an older version of sha3.h when FIPS is checked out, unfortunately that older version doesn't handle this.

Reply:
Double checking to be sure.

Reply:
Ope, I was wrong. Good catch, thank you! Updated user_settings.h to define `NO_OLD_WC_NAMES`.